apiserver/facades/client/modelmanager/modelmanager_test.go

```diff
@@ -20,7 +20,6 @@ import (
 	apiservererrors "github.com/juju/juju/apiserver/errors"
 	"github.com/juju/juju/apiserver/facade"
 	"github.com/juju/juju/apiserver/facades/client/modelmanager"
-	"github.com/juju/juju/apiserver/facades/client/modelmanager/mocks"
```

Comment:
Just the same aggregation comment as modelinfo_test for the singular skipped tests.